### ISSUE TITLE
Enhancement: Should the CLI report events emitted when calling functions? 

### ISSUE THREAD

**Issue by crazyrabbitLTC** [zeppelinos/zos#1104]
This is in relation specifically when calling a function when creating a contract. There doesn't seem to be an easy way to check that events were emitted as the CLI does not appear to report them. Should this be a feature? It might be convenient to know if events are emitted when calling the initializer.

**Comment by spalladino** [comment]
Definitely. It should also be straightforward to do, as most of the code to do it is in place for `send-tx`.
